calliope/backend/pyomo/model.py

```diff
@@ -101,32 +102,23 @@ def generate_model(model_data):
 
     # Constraints
     constraints_to_add = [
-        'energy_balance.load_constraints',
-        'dispatch.load_constraints',
-        'network.load_constraints',
-        'costs.load_constraints',
-        'policy.load_constraints',
-        'milp.load_constraints',
-        'group.load_constraints'
+        i.split('.py')[0] for i in os.listdir(constraints.__path__[0])
```

Comment:
`i.strip('.py') for i in os.listdir(constraints.__path__[0]) if i != '__init__.py'` would seem to do the same thing but a little less arcane?

Reply:
I decided to be more generic to account for the existence of any other non-constraint files in that folder (i.e. files can be hidden from being loaded if they are double underscored). Happy to be more explicit though, if you think this functionality is worthless.

Reply:
I would only specifically exclude `__init__.py` as the other behaviour might be unexpected. Unless there is a convention in Python to ignore double-underscored files?

Reply:
I also excluded `__pycache__`, since that shows up locally for me.